.gitignore

```diff
@@ -18,3 +18,7 @@ environment.sh
 
 # os files 
 .DS_Store
+
+*.plan
+*/.kitchen/**
+*.lock
```

Comment:
https://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore